test/rest-smoketest.sh

```diff
@@ -37,7 +35,16 @@ EOF
     exit 1
 }
 
-[[ -r "$COOKIES" ]] || error 
+[[ -z $GOAUTH  && -z $COOKIES ]] && COOKIES=${COOKIES:-cookie}
+[[ -n $COOKIES && ! -r $COOKIES ]] && error
+
+#Supported deployments: amazons3 or filesystem
+if [[ -z $DEPLOYMENT ]]
+then
+    DEPLOYMENT="filesystem"
+fi
+
+echo "Using $DEPLOYMENT deployment"
```

Comment:
This progress output should be to stderr, not stdout, so you need to add the `>&2` redirect as in existing printf statements further down